R/pkg/R/DataFrame.R

```diff
@@ -2635,7 +2635,7 @@ setMethod("write.df",
             write <- callJMethod(df@sdf, "write")
             write <- callJMethod(write, "format", source)
             write <- callJMethod(write, "mode", jmode)
-            write <- callJMethod(write, "save", path)
+            write <- callJMethod(write, "save", options)
```

Comment:
oops, sorry I broke it.
right, I think the right approach is 
```
write <- callJMethod(write, "options", options)
write <- callJMethod(write, "save", path)
```

`save` should be the last call.